config/opal_check_attributes.m4

```diff
@@ -556,6 +558,21 @@ AC_DEFUN([OPAL_CHECK_ATTRIBUTES], [
         ],
         [],
         [])
+
+    _OPAL_CHECK_SPECIFIC_ATTRIBUTE([optnone],
+        [
+        void __attribute__ ((__optnone__)) foo(void);
+        void foo(void) { return ; }
+        ],
+        [],
+        [])
+
+    _OPAL_CHECK_SPECIFIC_ATTRIBUTE([extension],
+        [
+         #define FOO __extension__ ({size_t bar; bar = 3;})
```

Comment:
I do not think that this check does anything.  It just tests to see whether `#define FOO ...` compiles or not, which, since it's just defining a preprocessor macro, will always succeed.